client/src/index.jsx

```diff
@@ -47,8 +47,8 @@ class Review extends Component {
 
   render() {
     return (
-      <div className="reviews-component">
-        <h1 className="reviews-component-header">REVIEWS</h1>
+      <div className="reviews-component container">
```

Comment:
did you mean to have a hyphen between component and container here?

Reply:
No container is a bootstrap class, I just shortened the classname that I chose because I thought it was too long.